templates/prometheus.yaml

```diff
@@ -72,10 +72,10 @@ spec:
         resources:
           # keep request = limit to keep this container in guaranteed class
           limits:
-            cpu: 100m
+            cpu: 200m
             memory: 100Mi
           requests:
-            cpu: 100m
+            cpu: 200m
```

Comment:
`100m`

Reply:
<img width="1067" alt="Screen Shot 2019-06-05 at 11 40 49 PM" src="https://user-images.githubusercontent.com/3602792/58992530-60a4fb00-87eb-11e9-90a2-8b35f6496698.png">

Reply:
this is the Cpu since the last time we deployed grafana... let's check if tomorrow we get a bigger spike when having more dashboards in grafana. But I assume we can remain with the current settings.

did you reproduce this alert again ? is it only happening when creating the cluster ? That could be negligible

Reply:
it happened twice that's all.

Reply:
we can try tomorro.w

Reply:
changed though.